Makefile

```diff
@@ -30,8 +29,9 @@ fmt:
 	go fmt
 
 .PHONY: docs
-docs:
-	tfplugindocs
+docs: build
+	$(MAKE) -C tfplugindocs_working generate-schema
+	tfplugindocs generate --providers-schema tfplugindocs_working/schema.json
```

Comment:
Now we generate the schema separately from generating the docs to allow us bypass the `tfplugindocs` auto compiling the provider.